Topic 0/src/point3/Factory.java

```diff
@@ -0,0 +1,16 @@
+package point3;
+
+public class Factory {
+	
+	public Hospital getHospital(String dimension){
+		Director director= null;
+		if (dimension.equals("small"))
```

Comment:
[Low] To avoid NullPointerException while comparing, it is a good practice to compare in this way:
<code>if ("small".equals(dimension))</code>
In this case, if dimension is null, the the condition is false but it doesn't throw an error.